package.json

```diff
@@ -21,6 +21,7 @@
     "prettier-project": "yarn prettier-write 'src/**/*.{ts,tsx}'",
     "prettier-write": "yarn prettier --write",
     "prettier": "prettier",
+    "release": "auto shipit",
```

Comment:
This will need calling in CI, but structurally this is sound 👍

Reply:
Yup, i was going to do a second pass and remove the semantic-release flow in favor of this